pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: flutter_blue_plus
 description: Flutter plugin for connecting and communicationg with Bluetooth Low Energy devices, on Android and iOS
-version: 1.4.0
+version: 1.4.1
```

```suggestion
version: 1.5.0
```
Hi, I'm just randomly reviewing your PR :smile:. I think version should be bumped to 1.5.0 because you change public API by changing getter's `isScanning` type from `Stream<bool>` to  `bool` (in `lib/src/flutter_blue_plus.dart`). This is breaking change. Also, I think this change should be mentioned **emphasized** in `CHANGELOG.md`.